lib/ftp.h

```diff
@@ -131,6 +131,10 @@ struct ftp_conn {
   bool cwdfail;     /* set TRUE if a CWD command fails, as then we must prevent
                        caching the current directory */
   bool wait_data_conn; /* this is set TRUE if data connection is waited */
+  /* newhost is the (allocated) IP addr or host name to connect the data
+     connection to */
+  unsigned short newport; /* connection to */
+  char *newhost;          /* this is the pair to connect the DATA... */
```

Comment:
These comments should be reversed but IMO are not needed at all because of the comment block above it

Reply:
Thanks, removing the per-field comments.